047-TrafficControlWithDapr/Student/Challenge-08.md

```diff
@@ -1,58 +1,21 @@
+**Attention Attendee:** Your coach will advise you on the path to take with this challenge.
 
-## Success Criteria
+### [Azure Kubernetes Service - Path A](./Challenge-08A.md)
 
-To complete this challenge, you must reach the following goals:
+- Deploy the Traffic Control App to Azure Kubernetes Service (AKS)
 
-- Validate that all 3 services are compiled into Docker images & stored in an Azure Container Registry.
-- Validate that you have successfully deployed all 3 services (`VehicleRegistrationService`, `TrafficControlService` & `FineCollectionService`) to an AKS cluster.
-- Validate that the local **Simulation** service runs & connects to your AKS-hosted services and that all the previous functionality still works (input messages, output messages, speeding violation emails, etc).
+### [Azure Container Apps - Path B](./Challenge-08B.md)
 
-## Tips
-
-- Change the host name for each service (in the `Program.cs` file) from `http://localhost` to `http://*` as this will allow the Kestrel server to bind to 0.0.0.0 instead of 127.0.0.1. This is needed to ensure the health probes work in Kubernetes.
-  - [Debugging K8S Connection Refused](https://miuv.blog/2021/12/08/debugging-k8s-connection-refused)
-- Use [ACR Tasks](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-tasks-overview) to simplify creation & deployment of the images to the registry.
-- Modify the `Resources/Infrastructure/Helm/dapr-trafficcontrol/values.yaml` with the specific connection strings, tenant IDs, registry names, etc for your deployment.
-- Use [Helm](https://helm.sh/docs/) to deploy all the AKS configuration files to Azure. Helm will substitute the values in the `values.yaml` file into the various configuration files.
-- Use various `kubectl` commands to validate that all the services are running in your AKS cluster. Here are some useful ones.
-
-  - Get all pods (in the current namespace)
-
-    ```shell
-    kubectl get pods
-    ```
-
-  - Describe a specific pod (to help debug deployment issues)
-
-    ```shell
-    kubectl describe pod <pod-name>
-    ```
-
-  - Tail the logs of a specific pod (and follow)
-
-    ```shell
-    kubectl logs fine-collection-service-7c99df4c85-2bxvf -f
-    ```
-
-## Learning Resources
-
-- [AKS Key Vault integration](https://learn.microsoft.com/en-us/azure/aks/csi-secrets-store-driver)
-- [AKS Workload Identity](https://learn.microsoft.com/en-us/azure/aks/workload-identity-overview)
+- Deploy the Traffic Control App to Azure Container Apps (ACA)
 
 Thanks for participating in these hands-on challenges! Hopefully you've learned about Dapr and how to use it. Obviously, these challenges barely scratch the surface of what is possible with Dapr. We have not touched upon subjects like: hardening production environments, actors, integration with Azure Functions and Azure API Management just to name a few. So if you're interested in learning more, I suggest you read the [Dapr documentation](https://docs.dapr.io).
```

Comment:
Do we want to remove this block since they are just choosing the path in challenge 8?  8A and 8B contain this when they have completed the challenge.

Reply:
good point, removed